v1.1-alpha.20170601.md

```diff
@@ -0,0 +1,80 @@
+- Some `make` targets have changed to follow convention. `make` by default now builds the binary without running tests. `make install` defaults to `/usr/local/bin` instead of `$GOPATH/bin` (use `make install prefix=/path/` to change). `make check` is now an alias for `make test`; the former `make check` is now called `make lint`. [#15909](https://github.com/cockroachdb/cockroach/pull/15909) [#16030](https://github.com/cockroachdb/cockroach/pull/16030)
+- The Kubernetes templates now use the latest production release instead of tracking testing releases. [#15878](https://github.com/cockroachdb/cockroach/pull/15878)
+
+### SQL Language Changes
+
+- The `information_schema.user_privileges` table is now supported. [#15745](https://github.com/cockroachdb/cockroach/pull/15745)
+- The `ON UPDATE RESTRICT` and `ON DELETE RESTRICT` modifiers can now be used when creating foreign keys (this behavior is the default, but now it can be specified explicitly). [#15815](https://github.com/cockroachdb/cockroach/pull/15815)
+- The hash functions `sha512()`, `fnv32()`, `fnv32a()`, `fnv64()`, `fnv64a()`, `crc32ieee()`, and `crc32c()` are now supported. All hash functions now accept multiple arguments and arguments of type `BYTES`. [#15828](https://github.com/cockroachdb/cockroach/pull/15828) [#15859](https://github.com/cockroachdb/cockroach/pull/15859)
+- Casts to type `DECIMAL` can now specify precision and scale. [#15834](https://github.com/cockroachdb/cockroach/pull/15834)
+- A new aggregate function `xor_agg()` is now available. [#15831](https://github.com/cockroachdb/cockroach/pull/15831)
+- Schema change commands now wait until the schema change is complete instead of starting it in the background. [#15598](https://github.com/cockroachdb/cockroach/pull/15598)
+- It is now possible to `SELECT` from the result of a `SHOW` statement with the syntax `SELECT ... FROM [ SHOW ... ] WHERE ...` [#15590](https://github.com/cockroachdb/cockroach/pull/15590)
+- It is now possible to `CREATE` a table and `ALTER` (or `DROP`) it in the same transaction. [#15929](https://github.com/cockroachdb/cockroach/pull/15929)
+- The standard SQL syntax `FETCH FIRST N ROWS ONLY` is now supported as an alias for `LIMIT`. [#16083](https://github.com/cockroachdb/cockroach/pull/16083)
+
+### Command-Line Interface Changes
+
+- A new `--advertise-port` flag is supported to go along with `--advertise-port`. [#15700](https://github.com/cockroachdb/cockroach/pull/15700)
+- A `raw` mode has been added to the SQL shell's `display_format` options. [#15590](https://github.com/cockroachdb/cockroach/pull/15590)
+- `cockroach zone set` with an incomplete config for the special system ranges now merges the given config with the default instead of using zeros. [#16048](https://github.com/cockroachdb/cockroach/pull/16048)
+- Nodes now validate their TLS certificates at startup to report errors more clearly. [#16051](https://github.com/cockroachdb/cockroach/pull/16051)
```

Comment:
None, the main change (certificate expiration metrics) is covered in the admin UI section.